code/modules/antagonists/villain/werewolf/werewolf_transformation.dm

```diff
@@ -1,8 +1,9 @@
 /datum/antagonist/werewolf/on_life(mob/user)
 	if(!user) return
-	var/mob/living/carbon/human/H = user
+	var/mob/living/carbon/human/H = user // the werewolf is still a human subtype
 	if(H.stat == DEAD) return
 	if(H.advsetup) return
+	if(H.mind?.has_antag_datum(/datum/antagonist/zombie)) return
```

Comment:
you should prob make it so WWs cant become zombies in the first place . dendors curse protects them